scripts/azdo/update_sstate_cache.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
```

Comment:
Note that I didn't add the `-u` option because I already run checks for arguments to print helpful error messages and the script starts to error out much earlier when the `-u` option is set.